pkgs/applications/networking/cluster/k9s/default.nix

```diff
@@ -19,7 +19,7 @@ buildGoModule rec {
     "-X github.com/derailed/k9s/cmd.date=1970-01-01T00:00:00Z"
   ];
 
-  tags = [ "netgo" ];
+  tags = [ "netcgo" ];
```

Comment:
This only happens on Darwin?

From time to time I use it behind a VPN, but mostly with public DNS, so I am not sure if it's reproducible on Linux.

Maybe an if for Darwin platform will be better?

If I remember correctly, there are a few problems with k9s and CGO.

Reply:
Honestly, my memory is fuzzy, sorry, so I can't point out the exact issue, just that I remember to basically do not use CGO. After searching at the tracker, it seems that there is a case for a few contexts, like OP and something for LDAP users (like https://github.com/derailed/k9s/issues/1895).

Will approve the PR, if something goes south, we revert it or add an if to the Darwin.